.github/workflows/codespell.yml

```
@@ -0,0 +1,19 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
+
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Codespell
+        uses: codespell-project/actions-codespell@v1
```

Use commit hash, upgrade to v2.
```suggestion
        uses: codespell-project/actions-codespell@94259cd8be02ad2903ba34a22d9c13de21a74461
```